examples/seismic/acoustic/operators.py

```diff
@@ -149,8 +150,8 @@ def GradientOperator(model, source, receiver, time_order=2, space_order=4, **kwa
 
     # Gradient symbol and wavefield symbols
     grad = Function(name='grad', grid=model.grid)
-    u = TimeFunction(name='u', grid=model.grid, save=True, time_dim=source.nt,
-                     time_order=2, space_order=space_order)
+    u = TimeFunction(name='u', grid=model.grid, save=save, time_dim=source.nt if save
```

Comment:
@mloubout why was ``save=True`` before? Was it just an oversight?

Reply:
It wasn't possible to calculate the gradient without having `u(..., save=True...)` before. This ability is being brought in by this PR (or was brought in by the previous PR, depending on how you want to look at it).

Reply:
oh I see, of course